.eslintrc.js

```diff
@@ -39,7 +39,9 @@ module.exports = {
             importNames: [
               'Dimensions', // Device
               'StyleSheet', // StyleSheet
-              'Text' // Text
+              'Text', // Text
+              'TouchableNativeFeedback', // TouchableItem
+              'TouchableOpacity', // TouchableItem
```

Comment:
Should `Button` and `TouchableHighlight` also be here? They are also button components from react native.

Reply:
You are right - updated. Check it, please.